src/app/components/Curation/index.tsx

```diff
@@ -14,13 +14,15 @@ import MessageBanner from '../MessageBanner';
 import idSanitiser from '../../lib/utilities/idSanitiser';
 import MostRead from '../MostRead';
 import { GHOST } from '../ThemeProvider/palette';
+import RadioScheduleContainer from '#app/legacy/containers/RadioSchedule';
```

Comment:
Can we rename this to RadioSchedule (we don't need to know that it's a Container when we're reading the code, and it keeps it consistent with the other components on this page)
```suggestion
import RadioSchedule from '#app/legacy/containers/RadioSchedule';
```